plugins/modules/acos_config.py

```diff
@@ -342,39 +363,33 @@ def main():
             module.fail_json(msg="Provided partition does not exist")
 
     diff_ignore_lines = module.params['diff_ignore_lines']
+    match = module.params['match']
     contents = None
     flags = 'with-default' if module.params['defaults'] else []
 
     startup_config_list = configuration_to_list(run_commands(module,
                                                              'show running-config'))
 
-    if module.params['file_path']:
-        try:
-            configuration_file = open(module.params["file_path"], 'r')
-            command_lines = configuration_file.readlines()
-            configuration_file.close()
-            run_commands(module, 'configure')
-            for line in command_lines:
-                if not line.startswith('!'):
-                    run_commands(module, line.strip())
-            run_commands(module, 'end')
-        except IOError:
-            module.fail_json(msg="File " + module.params["file_path"] + " Not Found!")
-
     if module.params['backup'] or (module._diff and
                                    module.params['diff_against'] == 'running'):
         contents = get_config(module, flags=flags)
         if module.params['backup']:
             result['__backup__'] = contents
-            backup(module, contents)
 
-    if module.params['lines']:
+    if any((module.params['lines'], module.params['src'])):

```

Comment:
Isn't a need to do this iteration here.

```python
if len(module.params['lines']) > 0 and len(module.params['src']) > 0:
```

`len()` does not iterate over the list. It simply looks up the information stored in memory.

Reply:
Yes. Updated that if statement with `if len(str(module.params['lines'])) > 0 and len(str(module.params['src'])) > 0`

Reply:
Resolved in PR#7